founder_story.md

```diff
@@ -0,0 +1,93 @@
+# Subtle.How
+
+# About the Founder
+
+My name is Kalin Naidoo, and I am a data founder.
+
+I am a database developer by trade and have been working as a database developer since 2014.
+I am creating a trusted workplace application for people of diverse backgrounds leveraging available disclosed data.
+
+With my professional domain expertise in database development, I think it will be something of value with sourcing the disclosed workforce data and standardizing it for Subtle customers to be able to compare companies against each other. I also have personal domain experience.
+
+# Personal Experience
+
+A couple of years ago, I took a job at a company where I experienced racial discrimination. My manager would pick on another diverse coworker for taking a 5-minute coffee break in the morning and began asking me to do things, not in my job description. 
```

Comment:
Minor nit, How about mentioning "At a previous job" than "A couple of years."?

Reply:
Sure thing. I am on kiddo duty until my wife gets home but I will update it tonight.

Reply:
Thank you, @KalinNaidoo Sonal Goyal from Zingg.ai submitted one just 20 minutes before yours. Otherwise, I would live to have started your founder story. I can relate to your experience, and I wish & want you to be successful in this mission.

Reply:
I appreciate you saying that. I would be honored to be your second data founder. Thank you so much for the opportunity to tell my story. I hope to reach others who can relate to my experience; that in itself is motivation to make the mission a reality.